docs/ShoppingContent-batchsource.md

```diff
@@ -0,0 +1,17 @@
+# Shopping Content Batch Source
+
+
+Description
+-----------
+This source reads product data from Google Shopping Content API.
+
+It outputs Products read from Shopping Content API in StructureRecord format, the output schema is aligned with Shopping
+Content Product definition. You can checkout the schema definition at productschema.json file.
```

Comment:
This document is for people using the UI, who won't know what productschema.json is. Should remove that sentence.

Reply:
There should also be a widgets file that is used to define how the properties should be rendered in the UI. In that file, you should be configuring a non-editable output schema, which will tell the user what this plugin outputs.

See https://github.com/cdapio/hydrator-plugins/blob/develop/spark-plugins/widgets/Twitter-streamingsource.json#L43 for an example.